### PR TITLE
feat: Added LiveFeed from Twitter

### LINKED ISSUE #1724: Get livefeed from Twitter
**Actual Behaviour**

There is currently only facebook feed. 

**Expected Behaviour**

The twitter social link could be used to get the live feed with the help from loklak.

**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

[milestone 10](https://github.com/fossasia/open-event-android/milestone/33)

### PR DESCRIPTION
**Pull request by dr0pdb**
Fixes #1724

Changes: Added LiveFeed from Twitter using the Loklak API.

Screenshots for the change: NA

@iamareebjamal Please review.

### CODE REVIEW

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Don't reuse the whole FeedFragment for both feeds. Follow Single Responsibility principle

**Comment by dr0pdb** [comment]
@iamareebjamal So should i create a new Fragment for it ?

**Comment by iamareebjamal** [comment]
Yes

**Comment by iamareebjamal** [comment]
Can you add a gif?

**Comment by dr0pdb** [comment]
videotogif_2018 01 29_19 04 04

**Comment by dr0pdb** [comment]
Note: The API right now only returns a single entry. [API call](https://api.loklak.org/api/search.json?q=fossasia&count=20&source=twitter)

I do understand that there is a lot of common code between FeedFragment and TwitterFeedFragment and also between it's layouts.
I propose to solve this in a new issue by creating a BaseFeedFragment class that will contain all the common code. Also the common code in layouts can be extracted out and used by ```include``` tags in the layout files.
I wanted to keep this PR centered at Twitter feed hence didn't change any thing in the FeedFragment.

**PS**
I missed the timestamp, i will add them and update the PR. Please suggest any other changes if needed.
For tweets having more than one images, i have used the first one. Any suggestions on that ?

**Comment by dr0pdb** [comment]
@mayank8318 @simarsingh24 Please review this PR specially for formatting errors. Since this is a relatively larger PR, I may have missed a few of them. 
Please review and let me know. :smile:  Thanks.

File: android/app/src/main/java/org/fossasia/openevent/adapters/TwitterFeedAdapter.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        super(twitterFeedItems);
+        this.context = context;
+        this.twitterFeedItems = twitterFeedItems;
+    }
+
+    @Override
+    public RecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View itemView = LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.item_twitter_feed, parent, false);
+
```
Comment (simarsingh24) [inline android/app/src/main/java/org/fossasia/openevent/adapters/TwitterFeedAdapter.java]: why two new lines here?

**Comment by simarsingh24** [comment]
@srv-twry the code is actually very well formatted couldn't find any formatting changes, good job! 👍

File: android/app/src/main/java/org/fossasia/openevent/adapters/TwitterFeedAdapter.java (hunk 1 above)
Comment (dr0pdb) [reply android/app/src/main/java/org/fossasia/openevent/adapters/TwitterFeedAdapter.java]: Yeah, will remove it. :+1:

**Comment by dr0pdb** [comment]
Changes: Added TimeStamp also


screenshot_2018-01-29-23-58-08-20


screenshot_2018-01-29-23-58-00-54


@iamareebjamal Please review.

File: android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -325,4 +326,8 @@
     <string name="about_fragment_to_date_event">To: %s</string>
     <string name="about_fragment_from_date_event">From: %s</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (iamareebjamal) [inline android/app/src/main/res/values/strings.xml]: Please remove this

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Good job

File: android/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dr0pdb) [reply android/app/src/main/res/values/strings.xml]: Yes, i have removed it. Saw it late.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**